src/plugins/intel_gpu/src/graph/graph_optimizer/mark_shape_of_subgraphs.cpp

```diff
@@ -78,6 +79,13 @@ bool mark_shape_of_subgraphs::can_mark_node(const program_node& node) {
             return false;
     }
 
+    // Exclude stride_slice primitive if it's input is big const ternsor, else CPU reference implementation
+    // will lead to huge performance drop.
+    if (node.is_type<strided_slice>() && node.get_dependency(0).is_constant() &&
+        node.get_dependency(0).get_output_layout().count() > 1024 * 1024) {
```

Comment:
Probably the allowed size of constant should be even lower, but I think we can go with current solution now